examples/react-app/src/index.tsx

```diff
@@ -82,14 +83,16 @@ ReactDOM.render(
         }}
       >
         <span>Menu Item</span>
-
         <PlmgSeparator
           thickness="thick"
           style={{ alignSelf: 'stretch' }}
           direction="vertical"
           color="#78909C"
         ></PlmgSeparator>
-        <span>Menu Item 2</span>
+        <span>Menu Item 2</span>{' '}
```

Comment:
This space does nothing. Furthermore, adding text spaces to try and separate your elements is very lazy and will be prone to error. Use css to position your elements.
```suggestion
        <span>Menu Item 2</span>
```